demos/mysql.yaml

```diff
@@ -0,0 +1,34 @@
+doc: |
+  Experimental example of using MySQL driver loaded at runtime.
+
+  Need to do 'go build -buildmode=plugin' in chans/sqlc/mysql to have
+  a chance of working.
+spec:
+  phases:
+    phase1:
+      steps:
+        - pub:
+            payload:
+              make:
+                name: sql
+                type: sql
+                config:
+                  drivername: "mysql"
+                  datasourcename: "user:password@/dbname"
+                  driverplugin: chans/sqlc/mysql/mysql.so
```

Comment:
Is this file supposed to be included in this PR? If not, it might be worth adding a timeout on the next receive in case people run this demo without having had created this file

Reply:
That `.so` shouldn't be in the PR (it's compiled), but docs in the spec suggest a way to build it.

Good idea re timeout. I'll add that.